glmmTMB/R/utils.R

```diff
@@ -732,6 +732,15 @@ nullSparseMatrix <- function() {
 }
 
 
+##' Check OpenMP status
+##'
+##' Checks whether OpenMP has been successfully enabled for this
+##' installation of the package. (Use the \code{parallel} argument
+##' to \code{\link{glmmTMBControl}}, or set \code{options(mc.core=[value])},
```

Comment:
`mc.cores` ?

Reply:
>  which steps/stages of the TMB run involve the Cholesky factorization?

Almost all steps: `obj$fn()` `obj$gr()` etc. In particular the newton steps and determinant calculations.

Reply:
Is there an intuition about when/for what kinds of models or data the Cholesky step will dominate?